src/tomodachi_testcontainers/clients/snssqs_client.py

```diff
@@ -47,15 +47,30 @@ def __init__(self, sns_client: SNSClient, sqs_client: SQSClient) -> None:
         self.sns_client = sns_client
         self.sqs_client = sqs_client
 
-    async def subscribe_to(self, topic: str, queue: str, attributes: Optional[Mapping[str, str]] = None) -> None:
+    async def subscribe_to(
+        self, topic: str, queue: str, fifo: bool = False, attributes: Optional[Mapping[str, str]] = None
+    ) -> None:
         """Subscribe a SQS queue to a SNS topic; create the topic and queue if they don't exist."""
         list_topics_response = await self.sns_client.list_topics()
         topic_arn = next((v["TopicArn"] for v in list_topics_response["Topics"] if v["TopicArn"].endswith(topic)), None)
+
+        topic_attributes: Mapping[str, str] = {}
+        queue_attributes: Mapping[QueueAttributeNameType, str] = {}
+        if fifo:
```

Comment:
**suggestion:** when tomodachi creates FIFO topics and queues, it sets `ContentBasedDeduplication` to `false`. I think we should do the same here